generator/internal/rust/annotate_test.go

```diff
@@ -39,6 +39,78 @@ func TestPackageNames(t *testing.T) {
 	}
 }
 
+func TestServiceAnnotations(t *testing.T) {
+	request := &api.Message{
+		Name:    "Request",
+		Package: "test",
+		ID:      ".test.Request",
+	}
+	response := &api.Message{
+		Name:    "Response",
+		Package: "test",
+		ID:      ".test.Response",
+	}
+	method := &api.Method{
+		Name:         "GetResource",
+		ID:           ".test.Service.GetResource",
+		InputTypeID:  ".test.Request",
+		OutputTypeID: ".test.Response",
+		PathInfo: &api.PathInfo{
+			Verb: "GET",
+			PathTemplate: []api.PathSegment{
+				api.NewLiteralPathSegment("/v1/resource"),
+			},
+		},
+	}
+	noHttpMethod := &api.Method{
```

Comment:
just curious: are we just testing that nothing breaks?

Reply:
No, I just forgot to add the assertion.

Reply:
PTAL